browser/playlist/playlist_service_factory.cc

```diff
@@ -123,7 +123,7 @@ PlaylistServiceFactory::PlaylistServiceFactory()
           "PlaylistService",
           BrowserContextDependencyManager::GetInstance()) {
   PlaylistDownloadRequestManager::SetPlaylistJavaScriptWorldId(
-      ISOLATED_WORLD_ID_CHROME_INTERNAL);
+      ISOLATED_WORLD_ID_BRAVE_INTERNAL);
```

Comment:
Continued from : https://github.com/brave/brave-core/pull/16419#discussion_r1066181617

We haven't decided yet if we should use a unique world id for playlist.

Reply:
Hi, @diracdeltas . I'm not sure what was the conclusion of the discussion. What is your take on this? and also wondering how this PR looks to you in general.

Reply:
@sangwoo108 let's leave it in `ISOLATED_WORLD_ID_BRAVE_INTERNAL` for now